plugins/repository-s3/build.gradle

```diff
@@ -236,9 +238,11 @@ task s3FixtureProperties {
       "s3Fixture.temporary_key"          : s3TemporaryAccessKey,
       "s3Fixture.temporary_session_token": s3TemporarySessionToken,
       "s3Fixture.ec2_bucket_name"        : s3EC2Bucket,
-      "s3Fixture.ecs_bucket_name"        : s3ECSBucket
+      "s3Fixture.ecs_bucket_name"        : s3ECSBucket,
+      "s3Fixture.disableChunkedEncoding" : s3DisableChunkedEncoding
   ]
 
+  outputs.upToDateWhen { false }
```

Comment:
This will make the build never can he anything  won't it?

I think I fixed the problem a little cleaner in https://github.com/elastic/elasticsearch/pull/44491 and wonder if we even any hack here once that's merged?

Reply:
I will test again after your change merged.